crawl-ref/source/travel.cc

```diff
@@ -2214,6 +2219,108 @@ static int _prompt_travel_branch(int prompt_flags)
     }
 }
 
+static level_pos _prompt_travel_altar()
+{
+    level_pos altar_pos;
+    extern map<level_pos, god_type> altars_present;
+
+    if (altars_present.empty())
+        return altar_pos;
+
+    while (true)
+    {
+        clear_messages();
+
+        int linec = 0;
```

Comment:
This should be `col` instead of `linec`; a fair amount of existing code uses `col` to mean "column".